.github/workflows/release.yml

```diff
@@ -0,0 +1,65 @@
+name: Publish to pub.dev
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
this should be `release` since there can be multiple merges to master before release if published

Reply:
Thanks, @singhashmeet. Yes, I agree, but we'll only publish if the `version` attribute on `pubspec.yml` is different between `master` and the local branch. The downside is that we will run it on every merge to master.

That being said, do you think we can use master?

Reply:
I'll change to release so we can avoid being charged every time someone merges to master.